Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:18.04
+
+RUN apt-get update \
+  && apt-get install -y python3-pip python3-dev \
+  && cd /usr/local/bin \
+  && ln -s /usr/bin/python3 python \
+  && pip3 install --upgrade pip \
+  && apt-get install libssl-dev \
+  && apt-get install -y locales
+RUN locale-gen en_US.UTF-8
+ENV LC_ALL=en_US.UTF-8
+ENV LANG=en_US.UTF-8
+COPY requirements-docker.txt /home/
+RUN pip install -r /home/requirements-docker.txt
+RUN pip install docker-compose
+COPY .env /home/
+COPY spin-docker.py /home/
+COPY docker-compose.yml /home/
+COPY createPemFiles.py /home/
+COPY web_s2i /home/web_s2i/
+COPY db_s2i  /home/db_s2i
+COPY app_s2i  /home/app_s2i
+CMD [ "/bin/bash" ]
```

Comment:
Why `CMD` and not `ENTRYPOINT`?

Reply:
What's the benefit of entrypoint here?

Reply:
All depends how you want the interaction with the container.
If you want that implicitly it will run the `spin.py`, you need to set it as entrypoing
If you fail to make it work, I suggest using
`ENTRYPOINT ["/usr/local/bin/python", "/home/spin-docker.py"]